### ISSUE TITLE
TAIR locus identifiers: ATGs vs numeric IDs

### ISSUE THREAD

**Issue by cmungall** [identifiers-org/registry#5]
Trying to reconcile this with GO

tair.locus is expected to be `^AT[1-5]G\d{5}$`

https://www.ebi.ac.uk/miriam/main/datatypes/MIR:00000050
https://github.com/identifiers-org/registry/blob/master/prefix/tair.locus.md

However, in GO, we use prefix TAIR and local IDs locus:2005496

https://github.com/geneontology/go-site/blob/73ee1c0dd6128e08b788dffbe0025eb1fd4c3c06/metadata/db-xrefs.yaml#L2845-L2867

We would like to use URIs such as http://identifiers.org/tair.locus/2005496 but these don't resolve.

There doesn't appear to be anything in id.org for the numeric IDs, just the At3g15890 accessions

cc @tberardini @tonysawfordebi @kltm

**Comment by tberardini** [comment]
Not all TAIR loci are of the format ^AT[1-5,M,C]G\d{5}$.  We annotate some unsequenced genomic loci which have names that don't conform to the sequenced loci.  This is why the identifier is the bare number locus:nnnnnn or gene:nnnnnn.  How can we (TAIR) work with identifiers.org on the numerical accessions so that all annotated objects can be resolved through their system?

**Comment by sarala** [comment]
Hi Chris and Tanya,

Thanks for pointing this out. I have fixed the pattern now. However, to be consistent with what we already have for tiar.gene, the URL looks like https://identifiers.org/tair.locus/Locus:2005496. 

I would like to explore your suggestion tair.locus/2005496. Is this how you would prefer to access this resource? What about tair.gene and tair.protein? Please let me know your thoughts.

**Comment by tberardini** [comment]
I don't have a preference for either tair.locus/2005496 or tair.locus/Locus:2005496.  Using the numerical id + the type (locus) covers our use case of having both sequenced and unsequenced loci.

**Comment by cmungall** [comment]
I would prefer tair.locus/nnnn

(no more MGI:MGIs!)

**Comment by jmcmurry** [comment]
+1 to tair.locus/nnnn

**Comment by cmungall** [comment]
Any further decision on this?

**Comment by tberardini** [comment]
Who needs to decide? Do we (TAIR) need to do anything?

**Comment by cmungall** [comment]
No, it's up to identifiers.org - or I can make a PR if that helps

**Comment by jmcmurry** [comment]
@cmungall Unfortunately, at the moment, identifiers.org prefix file is a one-way street; a PR would not be worthwhile. In time...

**Comment by sarala** [comment]
I have updated tair.locus record to support http://identifiers.org/tair.locus/2005496 or you could use the compact identifier form tair.locus:2005496. This will require everyone to change how they access tair.locus.

**Comment by sarala** [comment]
Regarding MGI - https://identifiers.org/MGI:2442292 works. If you are using the URL form it will be https://identifiers.org/mgi/MGI:2442292
